components/StyledUpdate.js

```diff
@@ -185,15 +185,15 @@ class StyledUpdate extends Component {
         </StyledTag>
         {editable && (
           <React.Fragment>
-            <Box mr={2} fontSize="12px">
-              <ActionButton onClick={this.toggleEdit} data-cy="toggleEditUpdate">
+            <Box ml={2} mr={2} fontSize="12px">
+              <StyledButton buttonSize="tiny" onClick={this.toggleEdit} data-cy="toggleEditUpdate">
                 {intl.formatMessage(this.messages[`${mode === 'edit' ? 'cancelEdit' : 'edit'}`])}
-              </ActionButton>
+              </StyledButton>
             </Box>
             <Box mr={2} fontSize="12px">
-              <ActionButton onClick={this.deleteUpdate}>
+              <StyledButton buttonSize="tiny" onClick={this.deleteUpdate}>
                 <FormattedMessage id="update.delete" defaultMessage="delete" />
```

Comment:
While we're at it, we should use the same capitalization than "Edit". After merging this change, you'll need to run `npm run build:langs` to update the language files and commit the result.

```suggestion
                <FormattedMessage id="update.delete" defaultMessage="Delete" />
```